packages/react-components/src/components/MessageThread.tsx

```diff
@@ -1202,7 +1210,7 @@ export const MessageThreadWrapper = (props: MessageThreadProps): JSX.Element =>
             /* @conditional-compile-remove(image-gallery) */
             onInlineImageClicked={onInlineImageClicked}
             /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
-            attachmentsMap={inlineAttachments}
+            attachmentsMap={inlineAttachments[messageProps.message.messageId]}
```

Comment:
can we do something like to avoid setting the props in 1184
```
attachmentsMap={inlineAttachments[messageProps.message.messageId] ?? {} }
```
since most message will not have image